### ISSUE TITLE
Session expiration in new UI

### ISSUE THREAD

**Issue by jasf-** [NEU-Libraries/scholar-onesearch#233]
The new UI is giving warnings and timing out sessions after about 30 minutes of inactivity, which I don't recall seeing happen in our current UI.  

Session timeouts happen regardless of login status (at least, they time out while *not* logged in).  

Is this desirable behavior?  If not, do we have ability to change or adjust it?

![sessionexpiry](https://cloud.githubusercontent.com/assets/4661920/21900579/5bf411e8-d8c2-11e6-8a5f-2ee5756fd9a2.png)

**Comment by jasf-** [comment]
Looked at documentation and the listserv but didn't see much mention of the session timeout (aside from conversation about changing the language of the message).  Despite the warning message, it doesn't appear that the sessions actually *do* time out - I've had windows open for hours that pop up the message but don't actually time out.

TL;DR - I'm closing this.

**Comment by jasf-** [comment]
Interestingly enough, it looks as though the session timeout is actually working in the August release.

**Comment by jasf-** [comment]
Sessions are timing out in 15 minutes.  Dunno if there a standard here, but @RPagani & I chatted about this and 15 min. seems a bit on the short side to us.   Can the session time be lengthened? @nestochan @gam5

**Comment by gam5** [comment]
Yes. https://knowledge.exlibrisgroup.com/Primo/Knowledge_Articles/Session_Timeout_parameter

**Comment by jasf-** [comment]
Great.  Any objection to bumping up timeout to 30 minutes or even longer?

**Comment by winthropo** [comment]
Set for 30 minutes right now. Are we certain it is logging out before that now?

Documentation does say "Note that If long timeout is defined, the sessions do not get deleted from the memory, and this might cause the server to get out of memory." Is that a concern?

![screen shot 2017-08-18 at 11 33 40 am](https://user-images.githubusercontent.com/4663357/29466007-453403f8-8409-11e7-818a-ae750db4f4bf.png)

**Comment by winthropo** [comment]
no objections.

**Comment by winthropo** [comment]
Fixed this in staging. Set to 30 now.
